docs/index.rst

```diff
@@ -11,15 +11,15 @@ PyVO lets you find and retrieve astronomical data available from archives that
 support standard `IVOA <http://www.ivoa.net/>`__ virtual observatory service
 protocols.
 
-* `Table Access Protocol (TAP) <http://www.ivoa.net/documents/TAP/>`_
+* `Table Access Protocol (TAP) <dal/index.html#table-access-protocol>`_
```

Comment:
Internal sphinx references would be nicer as cross-links than these hard wired URLs. I can help you with adding those once there is an approval for the text changes

Reply:
Thank you very much for the suggestion. I have changed the URLs into Sphinx references, please have a look if it is ok.

Reply:
Yes, the links look good now, thanks!